docs/source/install.rst

```diff
@@ -4,6 +4,98 @@
 Installing Volttron
 ===================
 
+Installing Linux Virtual Machine
+--------------------------------
+VOLTTRON requires a Linux system to run. For Windows users this will require a virtual machine (VM). This section will
```

Comment:
Let's put the first sentence of this paragraph as an introduction above "Installing Linux Virtual Machine"